### ISSUE TITLE
Last Post of 2021

### ISSUE THREAD

**Issue by utkzas** [shivangdubey/dumbbell#8]
Hello Team
Hope you all are doing fine. This would probably be the last post of 2021. 
First of all, congratulations on doing wonders throughout 2021, coming straight to the point, the members aced in a lot of things and worked hard to make 2021 significant
One of our member started with his internship as Python Developer Intern and then as a Research Intern and further continued 2022 Research Goals
Another member aced brilliantly well in Android development and put a lot of efforts to his development & coding skills and finally made 2021 worthwhile
Another member had a great time competing in coding competitions (CP) and finishing a lot of courses & gained great problem solving skills though so many of his experiences.
Another member tried our CP initially and then spent half of the year in WebD and also did an Internship.

We did a lot of mistakes, maybe couldn't complete some of our goals, and for some, the year was just like a quick wind

We tried at have a great exposure by contacting and getting guidance from people working at Myntra, Amazon, DE Shaw, ION and many more to make the most out of our college life. I guess now the Tier makes no sense after squeezing out every drop of our resources and trying our best to gather knowledge while others were busy playing games all day and scrolling insta aimlessly.

We tried doing together but this didn't work out well, we all got out of sync and that's completely OK!

We all did many mistakes, tried hard again and fall again. But didn't give up.
Then why give up at [this](https://leetcode.com)? 
We finally realised our own ways of doing DSA. But in the end one this we all will be doing is common i.e. Leetcode
2022 is the year of our Placements and is worth all our efforts which we had/had-not put in previous 3 years.
Now is the high time to raise the bar...
@rahulsain @Jai-Gaur-26  @shivangdubey @utkzas

**Comment by shivangdubey** [comment]
This is fantastic and has greatly inspired me. This year, we are very much looking forward to achieving our goals and expanding our knowledge. From the start, problem solving will be a top priority.
CC: @utkzas @rahulsain @Jai-Gaur-26

**Comment by rahulsain** [comment]
I agree with the above post, now that placement season is coming up in 6 months, we should now also focus on our problem-solving skills, and at least commit daily to maintain consistency.
@Jai-Gaur-26 @shivangdubey @utkzas @rahulsain

**Comment by utkzas** [comment]
What I've thought that, instead of creating random sheets we must follow Pep Level 2 questions on Nados. Every single question on nados level 2 is also on leetcode. so whenever we do that question, we submit the same solution to leetcode too, simultaneously.

there are added advantages too of doing those questions, video solutions, articles, etc.
we must check out nados and resume our leetcoding journey from here... 
what do you all think? @utkzas @rahulsain @shivangdubey @Jai-Gaur-26 
(Suggestions are welcome)

**Comment by shivangdubey** [comment]
Agreed, to be honest, I am not well versed with the platform, NADOS. But as you say it's nice, will follow that. Or maybe we can follow any sheet (Striver, Pepcoding).
